src/domains/events.ml

```diff
@@ -12,6 +12,7 @@ type t =
   | UpdateExpSplit of exp (** Used by expsplit analysis to evaluate [exp] on post-state. *)
   | Assert of exp
   | Unassume of {exp: CilType.Exp.t; uuids: string list}
+  | Poison of Queries.VS.t
```

Comment:
Poison is a rather generic name. I'm guessing this refers to the notion of clobbering?

Reply:
Yes, though clobbering is somehow related to variables being placed in registers, which is not necessary for it to be Undefined Behavior in the C standard.